test/debugInfo.fromasm

```diff
@@ -2,48 +2,39 @@
  (type $i32_=>_i32 (func (param i32) (result i32)))
  (type $i32_i32_=>_i32 (func (param i32 i32) (result i32)))
  (type $none_=>_none (func))
- (import "env" "memory" (memory $memory 256 256))
- (data (global.get $__memory_base) "debugInfo.asm.js")
- (import "env" "__memory_base" (global $__memory_base i32))
+ (import "env" "memory" (memory $1 256 256))
+ (data (global.get $gimport$0) "debugInfo.asm.js")
+ (import "env" "__memory_base" (global $gimport$0 i32))
```

Comment:
I haven't been able to get these names to survive the round trip. @kripken, is this expected to work or is Binaryen incapable of emitting some names?

Reply:
Ah, actually this is causing the current test failures. wasm2js only adds `__wasm_fetch_high_bits` if it sees a global named `i64toi32_i32$HIGH_BITS`, but global names are being blown away by the round trip pass.

Reply:
After discussing with @kripken, this was fixed by not running stack IR passes in wasm2js.